Message/RemoveProductMessage.php

```diff
@@ -13,6 +13,9 @@
 
 namespace Sulu\Bundle\SyliusConsumerBundle\Message;
 
+/**
+ * @internal
```

Comment:
when are these dispatched? 🙂  what is the idea behind making this internal?

Reply:
these are dispatched from the producer plugin https://github.com/sulu/SuluSyliusProducerPlugin/blob/0.x/src/Producer/TaxonMessageProducer.php

Reply:
okay i see - so there is a copy of these messages in both projects, right? should we keep them in the same location in both projects?

Reply:
thats not possible because of PSR-4 - in the future we could extract the messages into an own package - but currently we have no problems with that 😂